### PR TITLE
Update Analysis of Learning Rate for Deep Neural Networks.md

### PR DESCRIPTION
**Pull request by SZ559**
I just find out that one member is not in the ORIE4741 list. Please accept this change, thanks~

### CODE REVIEW

**Comment by sijiafan** [comment]
I want to confirm with you that Henry (hco23) is now doing the project with you, but not the other group "StockMarketAnalysis".

**Comment by SZ559** [comment]
Hi! He is working on both projects. He already discussed that with
professor Udell~

Best,
Shishi

**Comment by sijiafan** [comment]
Ok. Thanks for letting me know.
